src/vs/workbench/contrib/terminal/node/terminalProcess.ts

```diff
@@ -121,7 +125,7 @@ export class TerminalProcess implements ITerminalChildProcess, IDisposable {
 		}, 0);
 		// Setup polling
 		this._titleInterval = setInterval(() => {
-			if (this._currentTitle !== this._ptyProcess.process) {
+			if (this._currentTitle !== this._ptyProcess!.process) {
```

Comment:
If `this._ptyProcess` is passed into _setupTitlePolling we can avoid the `!`

Reply:
Done